assets/css/refills/_badges.scss

```diff
@@ -0,0 +1,21 @@
+$small-font-size: 0.75em;
```

Comment:
`0.75` should be written without a leading zero as `.75`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)